sdcm/scan_operation_thread.py

```diff
@@ -200,7 +201,9 @@ def __init__(self, **kwargs):
     def get_table_clustering_order(self) -> str:
         for node in self.db_cluster.nodes:
             try:
-                with self.create_session(node) as session:
+                with self.db_cluster.cql_connection_patient(node=node, user=self.username,
+                                                            password=self.password, connect_timeout=300) as session:
+                    session.default_consistency_level = ConsistencyLevel.ONE
```

Comment:
can you explain why we are doing CL.ONE  here ? (and add it as a comment)

Reply:
done